mem-07-l.xml

```diff
@@ -8190,7 +8190,7 @@ Marc Okrand has said that he prefers to call the puns or "Easter Eggs" in the Kl
       <column name="definition_zh_HK">災難</column>
       <column name="definition_pt">catástrofe</column>
       <column name="definition_fi">katastrofi</column>
-      <column name="synonyms"></column>
+      <column name="synonyms">{Qugh:n:1h}</column>
```

Comment:
This is already linked in the `notes` section. (In general, when something is in the notes section, it's excluded from fields like `synonyms` and `see_also`, but probably this hasn't been done consistently.)

Also, I would prefer if changes to translations to be isolated from other changes, so changes like these probably belong in a separate PR.

Reply:
Is there a reason you want this in `synonyms`, e.g., you're using that field to build a graph of relationships between entries or something?

Reply:
Yes, I was thinking of using boQwI' as a basis for a thesaurus with different world relations. My first application would be evaluating word vectors I have created. Another idea I have is to create a tool that could suggest related words that rhyme, to assist poem writing.

The problem with the notes section is that it is not machine-readable. Perhaps it could contain tags invisible to the boQwI' user like:

```xml
<column name="notes">This word and {Qugh:n:1h:synonym} are mostly interchangeable. To refer to something that in all likelihood one can't recover from (like the loss of {pIraqSIS:n}), {lot:n:nolink} is better.[2]</column>
```